crates/cheatcodes/spec/src/vm.rs

```diff
@@ -325,6 +329,10 @@ interface Vm {
     #[cheatcode(group = Evm, safety = Unsafe)]
     function warp(uint256 newTimestamp) external;
 
+    /// Gets the current `block.timestamp`
```

Comment:
Actually, can we also document why you should use this cheatcode here instead of `block.X` too? Also please end sentences with `.`